salesforce-gui/src/test/resources/features/personalInformation/edit_personalInformation.feature

```diff
@@ -2,15 +2,13 @@
 Feature: Edit Personal Information
 
   Background:
-    Given I log in to Salesforce with "Editable User" credentials
+    Given I log in to Salesforce with "Account Owner User" credentials
```

Comment:
I think that "Account Owner User" could be a different kind of user related to scenarios where we work with Accounts, please change to "Editable User" because should be different Users and add him to InitialsUsers.json file.